pyproject.toml

```diff
@@ -55,6 +51,11 @@ test = [
     "pytest",
     "pooch",
 ]
+gui = [
```

Comment:
Two things:

- we should update the installation instructions to reflect the changes; say we can use `pip install essreflectometry[gui]`...
- What about `pythreejs`? Instead of having a `gui` section, should we rename it to something like `[all]` or `[extra]` and include `pythreejs` in there as well?